### ISSUE TITLE
Double https:// in default configuration host

### ISSUE THREAD

**Issue by damienwebdev** [CyberSource/cybersource-rest-client-php#20]
# Bug

## Description

As a developer attempting to use the rest client, I tried to make a request with the default configuration and get the error:

```txt
API call to https://https://apitest.cybersource.com/flex/v1/keys/ failed: Could not resolve host: https
```

Note the duplicate `https://` at the beginning of the query url.

## Solution
Note the two following files. The `callApi` method appends an https:// to the host URL. To fix, either remove the appended `https://` from `callApi` or, perhaps more-backward compatibly  use a regex to conditionally append the `https://` IFF the host url does not contain https:// at the start of the string.

https://github.com/CyberSource/cybersource-rest-client-php/blob/7d623f9acbc526b9ee182fdf318e1c09321c7114/lib/ApiClient.php#L172

https://github.com/CyberSource/cybersource-rest-client-php/blob/7d623f9acbc526b9ee182fdf318e1c09321c7114/lib/Configuration.php#L98

## Notes
This probably needs unit test coverage as well.

**Comment by gnongsie** [comment]
Hi,
I've provided the fix for this. A pull request will be sent shortly.

**Comment by gnongsie** [comment]
Hi,
The fix for this has been released as part of v0.0.6.

Thank you for bringing this to our notice. I will be closing this issue for now. 

Feel free to open a new issue for any further problems.
